query_panel/main.js

```diff
@@ -43,8 +53,7 @@ window.addEventListener("DOMContentLoaded", async function () {
 const app = createApp({
   data() {
     return {
-      count: 0,
-      data: null,
+      eventData: null,
```

Comment:
We have been changing the name of this field lots of times. I'm not sure if the new name is better, why not keep the name data?